WordPress/gutenberg
# [RNMobile][Monorepo] Update from master and fix web e2e tests

Part of the [Monorepo effort](https://github.com/WordPress/gutenberg/pull/18508)

## Description
This PR aims to get web e2e tests working on the monore repo feature branch ([feat/import-gutenberg-mobile-no-squash](https://github.com/WordPress/gutenberg/tree/feat/import-gutenberg-mobile-no-squash)). It does so by updating from gutenberg `master` and gutenberg-mobile `develop`.

## How has this been tested?
This enables back travis CI on the `feat/import-gutenberg-mobile-no-squash` branch.
- Check that all tests are passing in this PR
- Check that the gutenberg demo is running fine locally and that there is no regression: `npm run native ios/android` and `npm run native start`

## Types of changes
- Update from master

## Checklist:
